python/__init__.py

```diff
@@ -401,9 +406,19 @@ def is_uno(self):
 
   def is_dos(self):
     return self.get_type() == Panda.HW_TYPE_DOS
+  
+  def is_red(self):
+    return self.get_type() == Panda.HW_TYPE_RED_PANDA
```

Comment:
do we really need an is_xxx function for each hw?

Reply:
Mmm, don't know.. Was following pattern of the lib